netlify.toml

```
@@ -15,13 +15,13 @@ HUGO_ENV = "production"
 HUGO_ENABLEGITINFO = "true"
 
 [context.deploy-preview]
-command = "make deploy-preview"
+command = "cd themes/docsy && git submodule update -f --init && cd ../.. && make deploy-preview"
 
 [context.branch-deploy]
-command = "make deploy-preview"
+command = "cd themes/docsy && git submodule update -f --init && cd ../.. && make deploy-preview"
```

@aidanranney Netlify fails to build with this syntax, so I wonder if we could try this:
```suggestion
command = "git submodule update --init --recursive && make deploy-preview"
```